specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -378,63 +674,360 @@
+        },
+        "duration": {
+          "description": "Duration of word articulation, in ticks.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "AnalyzeConversationTranscriptionPiiLROResult": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROResult"
+        }
+      ],
+      "properties": {
+        "results": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TranscriptionPiiResult"
+          }
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "TranscriptionPiiResults"
+    },
+    "TranscriptionPiiResult": {
+      "required": [
+        "conversationItems"
+      ],
+      "properties": {
+        "conversationItems": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TranscriptionPiiConversationItemResult"
+          }
+        },
+        "allOf": {
+          "$ref": "#/definitions/ConversationResult"
+        }
+      }
+    },
+    "TranscriptionPiiTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Transcription PII task.",
+      "properties": {
+        "piiCategories": {
+          "$ref": "#/definitions/TranscriptionPiiCategories"
+        },
+        "includeAudioRedaction": {
+          "type": "boolean",
+          "default": false
+        },
+        "showDetectedEntityTypes": {
+          "type": "boolean",
+          "default": false
+        },
+        "redactionSource":{
+          "type": "string",
+          "$ref": "#/definitions/ContentSource"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/TaskParameters"
+        },
+        {
+          "$ref": "#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "ContentSource": {
+      "type": "string",
+      "description": "Enumeration of supported content source types.",
+      "enum": [
+        "lexical",
+        "itn",
+        "maskedItn",
+        "text"
+      ],
+      "x-ms-enum": {
+        "name": "ContentSource",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationIssueResolutionSummarizationLROTask": {
+      "type": "object",
+      "description": "Task definition for Issue Resolution Summarization in conversation.",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/IssueResolutionSummarizationTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "IssueResolutionSummarizationTasks"
+    },
+    "IssueResolutionSummarizationTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for an Issue Resolution Summarization task.",
+      "properties": {
+        "summaryAspects": {
+          "type": "string"
```

Comment:
It might be a better approach to have a `summarizationKind` parameter instead of this aspect enumeration. 
The values can be: "issuesAndResolutions", "caseNotes", "actionItems", etc. 
There's no point of allowing the full matrix of aspect options here. Instead, a set of pre-defined scenarios is a more structured way of using the API.

Reply:
@mikaelsitruk to respond.